pennylane/ops/channel.py

```diff
@@ -81,8 +82,12 @@ def compute_kraus_matrices(gamma):  # pylint:disable=arguments-differ
         if not 0.0 <= gamma <= 1.0:
             raise ValueError("gamma must be in the interval [0,1].")
 
-        K0 = np.diag([1, np.sqrt(1 - gamma)])
-        K1 = np.sqrt(gamma) * np.array([[0, 1], [0, 0]])
+        # small number to add to probabilities to prevent dividing by
+        # zero in the gradient of sqrt
+        eps = 1e-14
```

Comment:
I changed it so that:
1. `qml.math` now has an attribute `eps`.
2. All channel ops now use `qml.math.eps` instead of defining their own numerical epsilon.
3. The user can modify this by setting e.g. `qml.math.eps = 1e-8`.

Not sure if this is the cleanest solution so would love to know your thoughts.

Reply:
I think that sounds great! :)